src/http_rest_api_handler.cpp

```diff
@@ -599,9 +602,23 @@ Status HttpRestApiHandler::processModelMetadataKFSRequest(const HttpRequestCompo
     return StatusCode::OK;
 }
 
+Status parseInferenceHeaderContentLength(HttpRequestComponents& requestComponents,
+    const std::vector<std::pair<std::string, std::string>>& headers) {
+    for (auto header : headers) {
+        if (header.first == "Inference-Header-Content-Length") {
+            requestComponents.inferenceHeaderContentLength = stoi32(header.second);
```

Comment:
When value is not convertible to integer does it take no value and therefore enters the next `if` block resulting in:
`return StatusCode::REST_INFERENCE_HEADER_CONTENT_LENGTH_INVALID;`
is that right?